components/core/layout/Banner.vue

```diff
@@ -1,19 +1,10 @@
 <template>
-    <div class="flex font-serif font-semibold w-full flex-1">
-        <div class="image hidden md:block w-full md:w-3/6 md:h-64">
-            <slot></slot>
+    <div class="bannercontainer">
+        <div class="mainstyle horizstyle">
+            <slot name="standard"></slot>
         </div>
-        <div
-            class="
-                flex flex-col
-                items-start
-                mx-8
-                w-full
-                text-xs
-                md:w-3/6 md:text-base
-            "
-        >
-            <slot name="text"></slot>
+        <div class="mainstyle vertstyle">
+            <slot name="standard"></slot>
```

Comment:
Can you explain what's the rationale behind this change?
I'll be providing a better solution to optimize tailwind's rwd soon. In the meantime, see if you can find some clues from this merged PR: https://github.com/pycontw/pycontw-frontend/pull/220.